jni/getroot.c

```diff
@@ -71,6 +71,9 @@ int modify_task_cred_uc(struct thread_info* __kernel info)
 		return 1;
 
 	tsp = malloc(sizeof(*tsp));
+	if (!tsp)
+		return 1;
```

Comment:
If you want it consistent, i'd leave the return 1 here. Its like this on other allocation too, and we can leave the function retval check as it is.
But i can switch it back, no problem.